.clang-tidy

```diff
@@ -4,6 +4,30 @@
 # this codebase and we do not intend to fix.  The disabled checks appearing
 # thereafter in a separate alphabetical list have yet to be triaged.  We may
 # fix their errors or recategorise them as checks we don't care about.
+#
+# Comments on the checks we have decided are not worthwhile:
+#
+# * cert-dcl21-cpp (postfix operator++ and operator-- should return const objects)
+# This is an unconventional code style, and conflicts with
+# readability-const-return-type.
+#
+# * cert-env33-c (calls to system, popen)
+# Unlikely to catch bugs, and using system is convenient for portability.
```

Comment:
System? Portable?

Reply:
It's not great, but it's more portable than `CreateProcess` ;).